### ISSUE TITLE
webdriver-auto-update 1.2.0 fails

### ISSUE THREAD

**Issue by pwannh** [competencytestlvl/webdriver_auto_update#20]
I am running python 3.12.0 on a MacBook M1 with macOS 12.7.1 I just reinstalled webdriver-auto-update and tried to run it. The first problem is this:

  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/webdriver_auto_update/chrome_app_utils.py", line 15, in __init__
    raise NotImplemented(f"Unidentified OS: '{platform}'")

Reason: chrome_app_utils.py line 8 contains "Darwin", which should be "darwin".
I fixed this manually.

Next problem:
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/webdriver_auto_update/webdriver_manager.py", line 150, in download_latest_version
    latest_driver_zip = wget.download(
                        ^^^^^^^^^^^^^^

Reason: webdriver_manager.py line 13 defines CHROMEDRIVER_BASE_URL to a value that leads to a 404 error.

Please look into this.

**Comment by competencytestlvl** [comment]
@pwannh 

Thank you for raising this issue. Hence, kindly refer the following commit  https://github.com/competencytestlvl/webdriver_auto_update/commit/ebed572ac146ff166291c08252fe3fe8680e89b3  on a separate branch https://github.com/competencytestlvl/webdriver_auto_update/tree/bugfix/issue-%2320/webdriver-auto-update-1.2.0-fails I have created, which aims to address the following issues below: 

1. The platform should be evaluated to "darwin" according to macOS specifications.
2. CHROMEDRIVER_BASE_URL has been updated to https://storage.googleapis.com/chrome-for-testing-public. Therefore, in your case, latest_driver_zip (chromedriver download file) would likely be at this time of writing: https://storage.googleapis.com/chrome-for-testing-public/121.0.6167.184/mac-arm64/chromedriver-mac-arm64.zip

Do let me know if this resolves your issues or if there are any more inconsistencies. 
If everything is resolved, this fix will be merged onto the main branch and a new release v1.2.1 shall be created.
